src/en/about-juju.md

```diff
@@ -41,11 +40,12 @@ in seconds, on any cloud or server or virtual machine.
 ## Can I use Juju with Puppet or Chef or Ansible?
 
 Yes! Puppet, Chef, and Ansible are great tools for writing configuration
-files. Juju works a layer above that by focusing on the service the
-application delivers, regardless of the machine on which it runs. So the Juju
-charm for an application includes (amongst other things) all the logic for
-writing configuration files for that application - that logic itself can be
-written in whatever language or tool the author of the charm prefers.
+files. Juju works a layer above that by focusing on the long term operations
+required to keep that software running over time, regardless of the machine on
+which it runs. So the Juju charm for an application includes (amongst other
+things) all the logic for writing configuration files for that
+application - that logic itself can be written in whatever language or tool
+the author of the charm prefers.
```

Comment:
similarly 

"Puppet and Chef are great tools for configuration management.

Juju provides a higher level of abstraction, supplying the tools to manage the full scope of operations beyond deployment and configuration management, regardless of the machine on which it runs.

One of the main advantages of Juju is its dynamic configuration ability, which allows you to re-configure services on the fly, add, remove, or change relationships between services, and scale in or out with ease, sharing the operational knowledge and making the most of the wider community."